ibm/service/project/resource_ibm_project.go

```diff
@@ -720,6 +720,17 @@ func resourceIbmProjectMapToProjectConfigDefinitionPrototype(modelMap map[string
 	if modelMap["settings"] != nil {
 		model.Settings = modelMap["settings"].(map[string]interface{})
 	}
+	if modelMap["members"] != nil {
```

Comment:
Did we defined or update the schema to have this argument?

Reply:
I have regenerated the code based on the latest API spec, now it has been removed, it was a dirty field